packages/generator-blueprints/src/models.ts

```diff
@@ -1825,6 +1828,12 @@ export type MutationAddRolesToTeamArgs = {
 };
 
 
+export type MutationAddRolesToTeamAssignmentArgs = {
```

Comment:
there seems to be a duplication between this file and: data-service-generator, packages/amplication-client/src/models.ts and the cli etc, i'd consider sharing them in one place.